schema_based_on_diagram.sql

```diff
@@ -41,4 +41,28 @@ CREATE TABLE Invoice_Items (
   treatment_id INT,
   FOREIGN KEY (invoice_id) REFERENCES Invoices(id),
   FOREIGN KEY (treatment_id) REFERENCES Treatments(id)
-);
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍 Just like I have made my logo which you can see in the image above ⬆️

Reply:
Thank you for your feedback, since it is a group project we opted not to use our personal logo on the READMe file.